ballista/core/src/execution_plans/shuffle_writer.rs

```diff
@@ -55,30 +55,7 @@ use datafusion::arrow::error::ArrowError;
 use datafusion::execution::context::TaskContext;
 use datafusion::physical_plan::repartition::BatchPartitioner;
 use datafusion::physical_plan::stream::RecordBatchStreamAdapter;
-use lazy_static::lazy_static;
 use log::{debug, info};
-use lru::LruCache;
-use parking_lot::Mutex;
-use std::num::NonZeroUsize;
-use std::sync::atomic::{AtomicUsize, Ordering};
-
-lazy_static! {
```

Comment:
This stuff is no longer relevant once task groups are implemented so I ripped it out